src/note.cpp

```diff
@@ -459,6 +459,47 @@ TransPitch Note::GetTransPitch()
     return TransPitch(pname, this->GetChromaticAlteration(), this->GetOct());
 }
 
+void Note::UpdateFromTransPitch(TransPitch tp)
+{
+    data_PITCHNAME pname = static_cast<data_PITCHNAME>(tp.m_pname + PITCHNAME_c);
+    this->SetPname(pname);
+
+    Accid *accid = this->GetDrawingAccid();
+    if (accid->HasAccidGes()) {
+        data_ACCIDENTAL_GESTURAL gestural = ACCIDENTAL_GESTURAL_NONE;
+        switch (tp.m_accid) {
+            case -2: gestural = ACCIDENTAL_GESTURAL_ff; break;
+            case -1: gestural = ACCIDENTAL_GESTURAL_f; break;
+            case 0: gestural = ACCIDENTAL_GESTURAL_NONE; break;
```

Comment:
Should probably be ACCIDENTAL_GESTURAL_n

Reply:
Yes, the meaning of "0" as a value for `tp.m_accid` is "natural".

Reply:
You might also want to add `ACCID_GESTURAL_fff` and `ACCID_GESTURAL_sss` to match the full range available in MEI (code excerpt might not show those if they are already there).

Reply:
[data.ACCIDENTAL.GESTURAL](https://music-encoding.org/guidelines/v4/data-types/data.accidental.gestural.html) does not support triple-flat or triple-sharp.

Reply:
I added a request to add triple sharp/flats to`@accid.ges`, so eventually it may be possible: https://github.com/music-encoding/music-encoding/issues/639